frontend/post/post_details.css

```diff
@@ -73,13 +78,13 @@
     }
 
     .deleted-post-icon {
-        font-size: 3.75em; 
-        color: white; 
+        font-size: 3.75em;
+        color: white;
         margin: 0;
     }
 
-    .delete-post-text {
-        text-align: center;
+    #delete-btn-text {
+        display: none;
```

Comment:
is it only applied for mobile?

Reply:
Yes, because only on mobile we need the extra space for the other buttons.

The class `.delete-post-test` was incorrectly removed. I'm adding it back.